pyproject.toml

```diff
@@ -40,6 +40,7 @@ dependencies = [
     "click",
     "datasets",
     "deepspeed",
+    "eval_type_backport",
```

Comment:
This is for pydantic working with `|` with older python versions right?

Reply:
Yep, correct, without 3.9 crashes in pydantic class creation